src/scenes/pt.cpp

```diff
@@ -29,18 +30,20 @@ Radiance trace(Scene const &scene, Ray const &wo, TraceContext &ctx) {
 
 void PathTracer::render(Scene const &scene, Camera const &cam, AppContext &ctx,
                         std::vector<Pixel> &image) {
-  size_t idx = 0;
 
   SamplerStd smp;
-  smp.init(ctx.frame + 1, idx);
+  smp.init(ctx.ptFrame + 1, 0);
 
   TraceContext tctx;
   tctx.app     = &ctx;
   tctx.sampler = &smp;
 
   Radiance avgChange   = Radiance::Zero();
   size_t changeSamples = 0;
-  for (auto &pixel : image) {
+
+#pragma omp parallel for
```

Comment:
Add reduction clauses for the average and changeSamples.
https://bisqwit.iki.fi/story/howto/openmp/#TheReductionClause_2

Reply:
Added clauses, can't use Vec3 for avgChange, had to separate variables.

Reply:
Just use a single float, since that is what we care a bout in the end. Move maxCoeff inside the loop and it should be close enough.